src/init/isAdminInit.js

```diff
@@ -0,0 +1,16 @@
+export var isAdminInit;
+
+isAdminInit = function(c) {
+    var t = this,
+        appManager = c.appManager,
+        requestManager = c.requestManager,
+        apiPath = appManager.getApiPath();
+
+    return {
+        baseUrl: appManager.getPath() + '/api/me/authorization/ALL', 
+        success: function(r) {
+            appManager.isAdmin = r;
+            requestManager.ok(this);
```

Comment:
Can you refresh me why this is needed? We are only checking authorization, no user group? Sorry for my bad memory...